test/org/sagebionetworks/bridge/services/TaskServiceMockTest.java

```diff
@@ -82,16 +89,26 @@ public void before() {
         when(userConsentDao.getUserConsent(any(String.class), any(StudyIdentifier.class))).thenReturn(consent);
         
         Map<String,DateTime> map = Maps.newHashMap();
-        TaskEventService taskEventService = mock(TaskEventService.class);
+        taskEventService = mock(TaskEventService.class);
         when(taskEventService.getTaskEventMap(anyString())).thenReturn(map);
         
         ScheduleContext context = createScheduleContext(endsOn);
         List<Task> tasks = TestUtils.runSchedulerForTasks(user, context);
-
+        
         taskDao = mock(DynamoTaskDao.class);
+        when(taskDao.getTask(anyString(), anyString())).thenAnswer(new Answer<Task>() {
+            @Override
+            public Task answer(InvocationOnMock invocation) throws Throwable {
```

Comment:
Minor: You can use a lambda to get rid of some of the boilerplate. For an example, see DynamoTaskDaoMockTest.mockQuery()